### DISCUSSION TITLE
Attribute missing for some classes.

### DISCUSSION THREAD

**Discussion by whbldhwj** [inducer/islpy#83] [Troubleshooting] [answered]
Hi,

I just installed islpy 2021.1. While I'm trying to call the method isl.Context.set_max_operations, I got the error message that the Context object has no such an attribute. I noticed that this function is added in the auto-generated file _isl.py, and inside the function _add_methods(). Do you know if there is anything I've missed or how to fix this issue?

Thanks a lot!
Jayden

**Comment by inducer** [comment, marked as answer]
It's actually `gen_wrap.py` (in the root directory of the repo) that reads the isl headers and tries to make sense of it. You can just run that and/or look with a debugger what it does about that function. Patches welcome to improve coverage.

**Reply by whbldhwj** [reply]
Thanks for your quick response! I've fixed it by adding the coverage of ctx.h in this file.
I've created a pull request for this change. #84
